src/rustup-utils/src/notifications.rs

```diff
@@ -20,8 +20,8 @@ pub enum Notification<'a> {
     DownloadFinished,
     NoCanonicalPath(&'a Path),
     ResumingPartialDownload,
-    UsingCurl,
     UsingReqwest,
+    UsingCurl,
```

Comment:
This delta seems odd

Reply:
Yeah I'm not sure why that delta is there, since in the actual source it's in the same place lol. I'll try to see what I can do to fix it.